lib/command/terraform/generate.rb

```diff
@@ -5,15 +5,26 @@ module Terraform
     class Generate < Base
       SUBCOMMAND_NAME = "terraform"
       NAME = "generate"
+      OPTIONS = [
+        app_option(required: true)
+      ].freeze
       DESCRIPTION = "Generates terraform configuration files"
       LONG_DESCRIPTION = <<~DESC
         - Generates terraform configuration files based on `controlplane.yml` and `templates/` config
       DESC
       WITH_INFO_HEADER = false
-      VALIDATIONS = [].freeze
 
       def call
         File.write(terraform_dir.join("providers.tf"), cpln_provider.to_tf)
+
+        templates.each do |template|
+          generator = TerraformConfig::Generator.new(config: config, template: template)
+
+          # TODO: Delete line below after all template kinds are supported
+          next unless %w[gvc identity].include?(template["kind"])
+
+          File.write(terraform_dir.join(generator.filename), generator.tf_config.to_tf, mode: "a+")
```

Comment:
We are using `mode: "a+"` because more identities can appear in loop and we want to be able to write additional configs to `identities.tf` - because of that we recreate  `terraform` directory (see `#terraform_dir`) each time `cpflow terraform generate` command invoked